src/Components/Product.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Product= ({category, image, description,price}) =>{
+
+    return (
+        <article>
+            <span>{category}</span>
+            <img src={image} alt={description}/>
+                <p>{description}</p>
+            <h4>{price}</h4>
```

Comment:
Het is handig om hier het euro teken toe te voegen voor de prijs. Dus `<h4>€{price}</h4>`. Het document met aandachtspunten raadt aan om hier ook ",-" achter de prijs toe te voegen maar dat lijkt mij zelf niet handig; dan kun je nooit iets achter de komma zetten bij de prijs.